attributes/network/9_ledger_account.adoc

```diff
@@ -0,0 +1,84 @@
+*   3-out-of-7.
+    Low-trust donation. 7 people have a private key, but any 3 can sign a transfer.
+*   1 or 3-of-4.
+    Distributed backup. The private key can be recovered with 3 of the 4 parts, which can be distributed into safes or friends.
+
+Some example of dripping wallet's use cases include;
+
+*   vesting schedule for employees, in tokens.
+*   rent or loan payments.
+*   automatic transfer of funds to an execution network as gas payment.
+
+== Requirements
+*   An account MUST be created with an owner.
+    An account without an owner should be considered deactivated and is safe to be removed.
+*   Accounts are referred to by their identities.
+*   A server creating an account MUST use a subresource identity of its own identity for the identity of the account, or a public key it owns.
+
+== Detailed Design
+The simplified design for this attribute is to add the basics for account management, and let features (separate attributes) decide how the account act.
+Each type of account features should be additional attributes for the network.
+A server might support multiple account features, and apply multiple compatible features to the same account.
+
+A new namespace is created through this attribute; `account`, which contains the following basic methods implemented by this attribute:
+
+* `account.create`.
+    Creates an account and sets its owner.
+* `account.info`.
+    Retrieves information related to an account.
+* `account.set`.
+    Sets information related to the account.
+* `account.delete`.
+    Deletes an account.
+    The identity may not be reusable.
+
+=== Information
+An account has metadata associated with it.
+For example, it can have a name.
+
+The information is set when creating, and can be retrieved using `account.info`, and updated when using `account.set`.
+
+=== Features
+Features are specified through network attributes, exposed by the network's `status`, and attribute IDs set in the information of an account.
+
+=== Ownership
+An account can have one or more owners.
+Depending on features, owners will have special access controls over the account.
+For example, by default only an owner can set information on an account, or delete it.
+
+Owners is the only required role for a ledger account.
+Any additional roles should be defined by features.
+
+=== Ledger-less Accounts
+Accounts do not require a ledger, nor do they have to be token holders.
+Features implemented by the server might require the server to be a ledger (for example, a drip feature).
+
+== Constraints
+The design has a few flaws that can be mitigated:
+
+* There is a lack of ACL and roles, outside of owners.
+  Although this is by design to keep the API surface small, it makes it harder later to add such designs without revisiting all combinations of features.
+* There is a limit of ~2 billions accounts per ledger identity as the specification says that accounts must be subresource identities.
```

Comment:
This is a 31-bit uint according to the specification.

Reply:
It's 31 bits because the 1 bit of the identity is the tag for the subresource identity. If we decide to make the identity larger then it would be possible to make arbitrary number of accounts.